e2e/setup/demo_app.js

```diff
@@ -0,0 +1,48 @@
+/*
+MIT License
+
+Copyright (c) 2021-present, Elastic NV
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+const http = require("http");
+const Koa = require("koa");
+const path = require("path");
+const koaStatic = require("koa-static");
+
+const { env } = require("../services");
+
+const DEMO_APP_DIR = path.join(__dirname, "../demo_app");
+
+const startDemoApp = () => {
+  return new Promise(resolve => {
+    const koaInstance = new Koa();
+    koaInstance.use(koaStatic(DEMO_APP_DIR));
```

Comment:
Nothing aganist using Koa and its sub modules. But keeping the complexity and scope simpler, we can use a simpler static server than bringing in a koa framework and its middlewares. https://github.com/elastic/synthetics-demo/blob/6fc9ae3388b1054e6ff795c500544c496e4782ee/todos/synthetics-tests/hooks.journey.ts#L10-L12

Reply:
Much better, thank you! I was looking exactly for this before I started using Koa, but a Google search only pointed to even bigger software, like Express and similar things. Thank you, I'm all for _less_ stuff :D